fcc-test-suite-template-master/index.html

```diff
@@ -25,20 +25,22 @@ <h1 id="title">Ronaldinho</h1>
       </div>
     </center>
     </div>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-
-    <!--
-
-    Hello Camper!
 
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
-    - The freeCodeCamp Team
+     <section class="main">
+    <h1 class="description">I am Muthomi <br> glad to see <br> you!</h1>
+    <p class="about">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <h2 class="connect">LET'S CONNECT</h2>
+    <ul class="icons">
+      <li><a href="https://github.com/Kalunge"><img src="./img/github.png" alt=""></a></li>
+      <li><a href="https://www.linkedin.com/in/muthomi-titus-295024181/"><img src="./img/linkedin.png" alt=""></a></li>
+      <li><a href="https://medium.com/@muthomititus"><img src="./img/medium.png" alt=""></a></li>
+      <li><a href="https://twitter.com/titus_muthomi"><img src="./img/twitter.png" alt=""></a></li>
+      <li><a href="#"><img src="./img//vector.svg" alt=""></a></li>
```

Comment:
You did not add an alt text. Alt text adds description to an image on a web page and should be used on all images.